src/components/EventDetails/EventDetails.js

```diff
@@ -103,22 +105,36 @@ type Props = {
   contactsSmartAddresses: ContactSmartAddressData[],
   inactiveAccounts: Accounts,
   supportedAssets: Asset[],
+  getRef?: () => Object,
+  getScrollOffset?: (number) => ScrollToProps,
+  getMaxScrollOffset?: (number) => number,
 }
 
+type State = {
+  containerHeight?: number,
```

Comment:
I don't think it's a good idea (react wise) to have optional state params as this could drive crazy react optimiser. Could we change this to `containerHeight: ?number,`?

Reply:
Updated :)